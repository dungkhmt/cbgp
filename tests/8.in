50 50
50 80
14 3
49 42
36 15
19 11
27 6
47 17
47 2
21 5
28 7
34 29
32 26
37 4
20 5
29 14
4 5
32 13
43 0
6 22
49 36
18 20
43 10
32 10
6 2
24 0
41 6
28 1
36 41
48 24
32 25
45 4
30 9
20 23
38 6
24 7
35 45
49 39
9 4
9 12
15 0
25 9
5 8
26 11
3 0
40 15
34 16
46 39
44 26
32 33
31 9
47 7
37 45
27 28
32 11
29 3
24 2
2 3
35 37
34 14
32 9
27 1
22 14
3 16
13 10
42 18
18 5
17 7
42 5
11 16
42 15
12 0
43 19
30 23
27 36
36 1
26 4
32 31
11 3
43 33
18 23
6 3
1
30 5 71 2 60 3 57 6 29 21 4 4 63 11 66 2 22 22 38 13 36 8 48 7 36 11 37 9 52 11 26 1 49 8 64 3 18 5 37 7 15 6 0 0 57 16 16 7 33 2 39 11 31 23 70 4 68 2 53 9 29 20 39 14 45 8 37 10 50 9 25 22 76 0 27 22 42 32 4 0 28 2 64 12 22 1 34 7 40 32 25 25 2 0 63 5 30 2 21 0