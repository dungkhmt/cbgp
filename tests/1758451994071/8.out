(14, 3), (49, 42), (36, 15), (19, 11), (27, 6), (47, 17), (47, 2), (21, 5), (28, 7), (34, 29), (32, 26), (37, 4), (20, 5), (29, 14), (4, 5), (32, 13), (43, 0), (6, 22), (49, 36), (18, 20), (43, 10), (32, 10), (6, 2), (24, 0), (41, 6), (28, 1), (36, 41), (48, 24), (32, 25), (45, 4), (30, 9), (20, 23), (38, 6), (24, 7), (35, 45), (49, 39), (9, 4), (9, 12), (15, 0), (25, 9), (5, 8), (26, 11), (3, 0), (40, 15), (34, 16), (46, 39), (44, 26), (32, 33), (31, 9), (47, 7), (37, 45), (27, 28), (32, 11), (29, 3), (24, 2), (2, 3), (35, 37), (34, 14), (32, 9), (27, 1), (22, 14), (3, 16), (13, 10), (42, 18), (18, 5), (17, 7), (42, 5), (11, 16), (42, 15), (12, 0), (43, 19), (30, 23), (27, 36), (36, 1), (26, 4), (32, 31), (11, 3), (43, 33), (18, 23), (6, 3), 
numberIntersection=0 
minAngle=0.09966865249116186 
sumAngle=57.67293161183132 
minEdgeLength=2.23606797749979 
sumEdgeLength=2705.1223637620715 
minDistance=0.09950371902099892 
sumDistance=242587.67285156954 

0.0000000000 0.0996686525 57.6729316118 2.2360679775 2705.1223637621 0.0995037190 242587.6728515695
0: (142, 177),
1: (55, 196),
2: (83, 174),
3: (87, 170),
4: (142, 118),
5: (196, 191),
6: (71, 149),
7: (67, 198),
8: (131, 0),
9: (122, 147),
10: (124, 169),
11: (104, 170),
12: (124, 150),
13: (123, 167),
14: (84, 136),
15: (138, 191),
16: (100, 166),
17: (76, 186),
18: (156, 178),
19: (119, 171),
20: (168, 171),
21: (194, 71),
22: (73, 147),
23: (163, 147),
24: (121, 192),
25: (118, 157),
26: (101, 109),
27: (66, 169),
28: (58, 198),
29: (95, 163),
30: (160, 147),
31: (118, 151),
32: (113, 163),
33: (120, 165),
34: (101, 162),
35: (164, 112),
36: (48, 200),
37: (153, 115),
38: (0, 0),
39: (197, 200),
40: (133, 185),
41: (70, 151),
42: (156, 191),
43: (134, 172),
44: (1, 0),
45: (123, 5),
46: (200, 200),
47: (64, 177),
48: (137, 191),
49: (156, 200),
142 177 55 196 83 174 87 170 142 118 196 191 71 149 67 198 131 0 122 147 124 169 104 170 124 150 123 167 84 136 138 191 100 166 76 186 156 178 119 171 168 171 194 71 73 147 163 147 121 192 118 157 101 109 66 169 58 198 95 163 160 147 118 151 113 163 120 165 101 162 164 112 48 200 153 115 0 0 197 200 133 185 70 151 156 191 134 172 1 0 123 5 200 200 64 177 137 191 156 200 
0: (142, 177), 1: (55, 196), 2: (83, 174), 3: (87, 170), 4: (142, 118), 5: (196, 191), 6: (71, 149), 7: (67, 198), 8: (131, 0), 9: (122, 147), 10: (124, 169), 11: (104, 170), 12: (124, 150), 13: (123, 167), 14: (84, 136), 15: (138, 191), 16: (100, 166), 17: (76, 186), 18: (156, 178), 19: (119, 171), 20: (168, 171), 21: (194, 71), 22: (73, 147), 23: (163, 147), 24: (121, 192), 25: (118, 157), 26: (101, 109), 27: (66, 169), 28: (58, 198), 29: (95, 163), 30: (160, 147), 31: (118, 151), 32: (113, 163), 33: (120, 165), 34: (101, 162), 35: (164, 112), 36: (48, 200), 37: (153, 115), 38: (0, 0), 39: (197, 200), 40: (133, 185), 41: (70, 151), 42: (156, 191), 43: (134, 172), 44: (1, 0), 45: (123, 5), 46: (200, 200), 47: (64, 177), 48: (137, 191), 49: (156, 200), 
numberIntersection=0 minAngle=0.09966865249116186 sumAngle=57.67293161179338 minEdgeLength=2.23606797749979 sumEdgeLength=2705.1223637620656 minDistance=0.09950371902099892 sumDistance=242587.6728515673 
Time taken: 36.861 s
